test_case_results.js

```diff
@@ -0,0 +1,314 @@
+/******************************************************************************/
+/* TestCaseResults                                                            */
+/******************************************************************************/
+
+TestCaseResults = function (test_case, onEvent, onException, stop_at_offset) {
```

Comment:
Wouldn't be enough if you would restore `Tinytest` to `Packages.tinytest.Tinytest` after you loading the `tinytest.js`? So just create a small `.js` file and add it after you add `tinytest.js` with this content:

```javascript
Tinytest = Packages.tinytest.Tinytest;
```